Draft/draft-ietf-rtcweb-stun-consent-freshness.xml

```diff
@@ -252,10 +261,14 @@
         (i.e., a candidate pair has been marked as Succeeded), consent MUST be
         maintained following the procedure described in this document.</t>
 
-        <t>An endpoint MUST NOT send data other than paced STUN connectivity
-        checks or responses toward any transport address unless the receiving
-        endpoint consents to receive data. That is, no application data (e.g.,
-        RTP or DTLS) can be sent until consent is obtained.</t>
+        <t>An endpoint MUST NOT send data toward any transport address unless
+        the receiving endpoint has consented to receive data, unless the
+        messages are those that are used to establish consent. Connectivity
+        checks that are paced as described in Section 16 of <xref
+        target="RFC5245"></xref> and responses to connectivity checks are
+        permitted. That is, no application data (e.g., RTP or Datagram
+        Transport Layer Security (DTLS)) can be sent until consent is
+        obtained.</t>
```

Comment:
Are these my changes?  I don't see my commits in your revision history, so you must be doing something strange with version control.